tools/crm_simulate.c

```diff
@@ -1010,28 +1003,31 @@ main(int argc, char **argv)
     pe__set_working_set_flags(data_set, pe_flag_stdout);
     cluster_status(data_set);
 
-    if (quiet == FALSE) {
+    if (!out->is_quiet(out)) {
         int opts = options.print_pending ? pe_print_pending : 0;
 
         if (pcmk_is_set(data_set->flags, pe_flag_maintenance_mode)) {
-            quiet_log("\n              *** Resource management is DISABLED ***");
-            quiet_log("\n  The cluster will not attempt to start, stop or recover services");
-            quiet_log("\n");
+            printed = out->message(out, "maint-mode", data_set->flags);
         }
 
         if (data_set->disabled_resources || data_set->blocked_resources) {
-            quiet_log("%d of %d resource instances DISABLED and %d BLOCKED "
-                      "from further action due to failure\n",
+            PCMK__OUTPUT_SPACER_IF(out, printed == pcmk_rc_ok);
+            out->info(out, "%d of %d resource instances DISABLED and %d BLOCKED "
+                      "from further action due to failure",
                       data_set->disabled_resources, data_set->ninstances,
                       data_set->blocked_resources);
+            printed = pcmk_rc_ok;
         }
 
-        quiet_log("\nCurrent cluster status:\n");
+        PCMK__OUTPUT_SPACER_IF(out, printed == pcmk_rc_ok);
+        out->begin_list(out, NULL, NULL, "Current Cluster Status");
```

Comment:
pcs currently looks for just two literal strings in crm_simulate output, "Current cluster status:" and "Transition Summary:". Let's leave them the same, maybe with a comment as to why it's different.

https://github.com/ClusterLabs/pcs/blob/7d04456984ef5c49edcffb799107732402877e1d/pcs/resource.py#L3195

Reply:
Good catch, fixed.